sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -263,8 +263,7 @@ class SessionCatalog(
           CatalogColumn(
             name = c.name,
             dataType = c.dataType.catalogString,
-            nullable = c.nullable,
-            comment = Option(c.name)
```

Comment:
Like the existing master build, we removed this useless `comment` attribute. The major reason is the [schema comparison](https://github.com/apache/spark/blob/6d86403d8b252776effcddd71338b4d21a224f9b/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala#L163) also checks the comment.